packages/react-email/src/cli/utils/preview/hot-reloading/create-dependency-graph.ts

```diff
@@ -245,7 +245,7 @@ export const createDependencyGraph = async (directory: string) => {
   return [
     graph,
     /**
-     * @param pathToModified - A path relative to the previosuly provided {@link directory}.
+     * @param pathToModified - An absolute path to the module involved in this change
```

Comment:
Changed this comment because it was plain wrong before. The paths here were relative back when it was written, but I ended up changing this along the way.

Reply:
can we actually remove the comment? precisely to avoid having to change it later on? 😅

Reply:
Removed it 😂